qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -37,41 +43,80 @@ def _get_shared_links_for_study(study):
     return ", ".join(shared)
 
 
-def _build_study_info(studytype, user=None):
+def _build_study_info(studytype, user, results=None):
     """builds list of namedtuples for study listings"""
```

Comment:
Update the docstring (namedtuples no longer apply). Although this function is private, I think having a bit of numpy doc will help future developers...